hivemq/assets/dashboards/hivemq.json

```diff
@@ -0,0 +1,682 @@
+            "legend": false,
+            "tile_def": {
+                "custom_links": [],
+                "requests": [
+                    {
+                        "q": "sum:hivemq.networking.connections.current{*} by {host}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "area"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Current Connections",
+            "type": "timeseries",
+            "width": 33,
+            "x": 23,
+            "y": 8
+        },
+        {
+            "height": 8,
+            "tile_def": {
+                "autoscale": true,
+                "custom_links": [],
+                "precision": 0,
+                "requests": [
+                    {
+                        "aggregator": "last",
+                        "q": "sum:hivemq.subscriptions.overall.current{*}.as_count()"
+                    }
+                ],
+                "viz": "query_value"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Subscriptions",
+            "type": "query_value",
+            "width": 21,
+            "x": 1,
+            "y": 30
+        },
+        {
+            "height": 14,
+            "legend": false,
+            "tile_def": {
+                "custom_links": [],
+                "requests": [
+                    {
+                        "q": "sum:hivemq.networking.connections_closed.graceful.count{*} by {host}.as_count()",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "bars"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "Graceful Disconnects",
+            "type": "timeseries",
+            "width": 33,
+            "x": 23,
+            "y": 42
+        },
+        {
+            "bgcolor": "vivid_yellow",
```

Comment:
According to the Best Practices document, these label should have a grey background (although it's cool to match the logo colors)